dashboards/resources/node.libsonnet

```diff
@@ -20,7 +20,7 @@ local template = grafana.template;
       template.new(
         name='node',
         datasource='$datasource',
-        query='label_values(kube_pod_info{%(clusterLabel)s="$cluster"}, node)' % $._config.clusterLabel,
+        query='label_values(up{%(kubeStateMetricsSelector)s, %(clusterLabel)s="$cluster"}, node)' % $._config,
```

Comment:
@paulfantom @discordianfish  this completely broke this `Kubernetes / Compute Resources / Node (Pods)` dashboard.  Note that, this template query here for this node dashboard is expected to return all nodes in the K8s cluster.  kubeStateMetricsSelector(job="kube-state-metrics") above wouldn't return any node, as that series don't have any `node` label.

Reply:
You are right, would you like to create a PR with the fix?